pkg/networkmanager/wireguard.jsx

```diff
@@ -98,6 +103,7 @@ export function WireGuardDialog({ settings, connection, dev }) {
             const objpath = connection[" priv"].path;
             const [result] = await model.client.call(objpath, "org.freedesktop.NetworkManager.Settings.Connection", "GetSecrets", ["wireguard"]);
             setGeneratedPrivateKey(result.wireguard["private-key"].v);
+            setPeers(oldPeers => oldPeers.map((oldPeer, index) => ({ ...oldPeer, presharedKey: result.wireguard.peers?.v[index]["preshared-key"]?.v ?? '' })));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19521-20231024-110617-ce68b93e-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)